lib/api/websiteGet.js

```diff
@@ -9,6 +9,36 @@ import validateHeaders from '../utilities/validateHeaders';
 import { pushMetric } from '../utapi/utilities';
 
 /**
+ * _extractRedirectInfo - extract information from location for a redirect
```

Comment:
_extractRedirectInfo - convert url saved from x-amz-website header to same format as 
redirectInfo saved from a put bucket website configuration